io/module_wrt_grid_comp.F90

```diff
@@ -548,6 +548,28 @@ subroutine wrt_initialize(wrt_comp, imp_state_write, exp_state_write, clock, rc)
             latPtr(i,j) = lat(j)
```

Comment:
Not strictly part of this PR, but I would suggest instead of recomputing lonPtr(i,j) (line 547) to use already computed values from array lon (line 541). BTW, for consistency, the index in that loop (540-542) should be `i`, not `j`.